src/data/javascript-quiz.ts

```diff
@@ -1422,6 +1422,15 @@ const javascriptQuiz = [
       "JavaScript will concatenate all the strings together and attempt to convert 'cteri' to a number using the unary plus operator, and since 'cteri' is not a number, the result will be NaN(Not a Number) which altogether is banana",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus",
   },
+   {
+    Question: "Which of the following javascript methods is not an array method?",
+    Answer: ".search()",
+    Distractor1: ".includes()",
+    Distractor2: ".forEach()",
+    Distractor3: ".search()",
+    Explanation: "The search() method executes a search for a match between a regular expression and this String object.",
```

Comment:
We want to avoid using direct quotes from 3rd party sites like MDN. Let's rewrite to the following: 

```suggestion
    Explanation: "The search() method searches through the string to find a match to a regular expression. If no match is found, then the return value will be -1.",
```